contrib/Findsodium.cmake

```diff
@@ -55,23 +55,35 @@ if (UNIX)
     endif()
 
     if(sodium_USE_STATIC_LIBS)
+        foreach(_libname ${sodium_PKG_STATIC_LIBRARIES})
+            if (NOT _libname MATCHES "^lib.*\\.a$") # ignore strings already ending with .a
+                list(INSERT sodium_PKG_STATIC_LIBRARIES 0 "lib${_libname}.a")
+            endif()
+        endforeach()
+        list(REMOVE_DUPLICATES sodium_PKG_STATIC_LIBRARIES)
+
         # if pkgconfig for libsodium doesn't provide
         # static lib info, then override PKG_STATIC here..
-        if (NOT DEFINED sodium_PKG_STATIC_LIBRARIES)
+        if (sodium_PKG_STATIC_LIBRARIES STREQUAL "")
             set(sodium_PKG_STATIC_LIBRARIES libsodium.a)
         endif()
+
         set(XPREFIX sodium_PKG_STATIC)
     else()
+        if (sodium_PKG_LIBRARIES STREQUAL "")
+            set(sodium_PKG_IBRARIES sodium)
```

Comment:
Should be set(sodium_PKG_**L**IBRARIES sodium)

Reply:
fixed 🙈